### ISSUE TITLE
Create more layer-control sub-components

### ISSUE THREAD

**Issue by allthesignals** [NYCPlanning/labs-zola#93]
There are a few ways we will need to be able to control a layer in terms of filtering. So far, we have:

- [x] layer-timeline-control

I can see us needing these:
- [x] layer-multiselect-control
   This could have a dropdown select option or a list of checkboxes
- [x] layer-integer-range
   Much like layer-timeline-control, but the value is an integer rather than a date type. Could re-use (but prefer duplication over the wrong abstraction ;))
- [x] layer-static-legend

**Comment by allthesignals** [comment]
*layer-multiselect-control*

The multi-select could have a few API interfaces... I'm not sure how to approach this first. Either the layer-control can find the unique possible values from a column but then that marries the project to Carto. It's also less flexible. 

Another approach is to explicitly set the list of controls through an array of objects:
```javascript
[{ label: 'C1-1',
value: 'C1-1' },
{ label: 'C1-2',
value: 'C1-2' },
{ label: 'C1-3',
value: 'C1-3' },
{ label: 'C1-4',
value: 'C1-4' },
{ label: 'C1-5',
value: 'C1-5' },
{ label: 'C2-1',
value: 'C2-1' },
{ label: 'C2-2',
value: 'C2-2' },
{ label: 'C2-3',
value: 'C2-3' },
{ label: 'C2-4',
value: 'C2-4' },
{ label: 'C2-5',
value: 'C2-5' }]
```

Sometimes these categories are explicitly defined in the Mapbox layer-group config, sometimes not. For example, Zoning Districts defines a color for each "stop". 

A third approach is to allow this to be more explicit through more sub-components: 
```hbs
{{#map.layer-group config=commercialOverlays qps=qps as |group|}}
  {{group.checkbox-control value='C2-4' label='C2-4'}}
  {{group.checkbox-control value='C1-2' label='C1-2'}}
  //...
{{/map.layer-group}}
```

**Comment by allthesignals** [comment]
Closed by #109
